File: Ejercicios/Java/Clases/Libro.java
public class Libro {
    private String titulo;
    private String autor;

    public Libro() {
        this("Sin título", "Anónimo");
    }

    public Libro(String titulo, String autor) {
        this.titulo = titulo;
        this.autor = autor;
    }

    public Libro(String titulo) {
        this(titulo, "Desconocido");
    }

    public Libro(Libro otro) {
        this(otro.titulo, otro.autor);
    }
}

File: Ejercicios/C#/Clases/Libro.cs
public class Libro
{
    public string Titulo;
    public string Autor;

    public Libro()
    {
        Titulo = "Sin título";
        Autor = "Anónimo";
    }

    public Libro(string titulo, string autor)
    {
        Titulo = titulo;
        Autor = autor;
    }

    public Libro(string titulo)
    {
        Titulo = titulo;
        Autor = "Desconocido";
    }

    public Libro(Libro otro)
    {
        Titulo = otro.Titulo;
        Autor = otro.Autor;
    }
}
